docs/examples/git-repo/README.md

```diff
@@ -0,0 +1,84 @@
+## Usage 
+This example is modified on the basis of kustomized controller provided by flux.
+The kustomize-controller is part of a composable GitOps toolkit and depends on source-controller 
+to acquire the Kubernetes manifests from Git repositories and S3 compatible storage buckets.
+
+### 1. Install toolkit controllers
+Download the flux CLI:
```

Comment:
Another problem we really need to solve is how to handle these addons (including cloud service account/configs). As a super glue, manually install everything you want to glue is disappointing.

Reply:
yes, have designed a better solution with @hongchaodeng , will finish the design and share it ASAP.

Reply:
We have designed the initializer for it \cc @yangsoon